code/obj/machinery/computer/card.dm

```diff
@@ -38,6 +38,16 @@
 	icon = 'icons/obj/computerpanel.dmi'
 	icon_state = "id2"
 
+/obj/machinery/computer/card/portable
+  name = "portable identification computer"
+  icon_state = "idportC"
+
+  New()
+    ..()
+    src.AddComponent(/datum/component/foldable, /obj/item/objBriefcase/idcomputercaseC)
```

Comment:
Missed the mark here. Going to suggest `/obj/item/objBriefcase/blue_stripe` as the path name. The briefcase has no bearing on its contents.